cubedash/summary/_stores.py

```diff
@@ -1561,6 +1564,34 @@ def get_dataset_footprint_region(self, dataset_id):
             row.region_code,
         )
 
+    def get_dataset_uris(self, dataset: Dataset):
+        """
+        Get a representation of the data URIs so that they
+        can be linked to to download them.
+        """
+
+        uri_list = {}
+
+        if is_doc_eo3(dataset.metadata_doc):
+            dataset_doc = serialise.from_doc(dataset.metadata_doc, skip_validation=True)
+            dataset_location = (
+                dataset_doc.locations[0] if dataset_doc.locations else None
+            )
```

Comment:
When running everything's coming back as a relative path (which are broken as they're relative to Explorer's webpage url). Presumably it's because this field is null.

(The raw indexed `dataset.metadata_doc` column doesn't contain locations)

We'll need to grab locations from the original Dataset object instead:
```suggestion
            dataset_location = (
                dataset.uris[0] if dataset.uris else None
            )
```